WordPress/src/main/res/layout/activity_log_type_filter_fragment.xml

```diff
@@ -1,13 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/coordinator_layout"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:layout_height="match_parent"
+    android:orientation="vertical">
 
-    <com.google.android.material.textview.MaterialTextView
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appbar_main"
+        style="@style/WordPress.AppBarLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:liftOnScrollTargetViewId="@+id/recycler_view">
+
+        <com.google.android.material.appbar.MaterialToolbar
+            android:id="@+id/toolbar_main"
+            style="@style/WordPress.ToolBar"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content" />
+
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
Minor: I am not sure why is that, but on my IDE, when reformat file is run on this layout it moves the `RecyclerVIew` on top of the `AppBarLayout`. Can you try it on yours as well?

If that is the case maybe it is better to go with what reformat suggests so that future reformats become easier.

Reply:
Yeah, I noticed this while working on the feature. I wanted to keep the order consistent with other views. But I agree that having to format the file manually is probably worse than having it a bit inconsistent with the rest. Tbh I'm not sure why the formatter prefers this order 🤷 . Fixed in 3e1d186f208a8981f3894e5b2039605f0636e4c6